docs/reference/commandline/login.md

```diff
@@ -23,8 +23,9 @@ If no server is specified, the default is defined by the daemon.
 
 Options:
       --help              Print usage
-  -p, --password string   Password
+  -p, --password string   Password (deprecated, use --password-stdin)
   -u, --username string   Username
+      --password-stdin    Read password from stdin
```

Comment:
Also, can you add an example on how to use this flag in the examples section below?